spikeinterface/widgets/matplotlib/unit_waveforms.py

```diff
@@ -32,6 +32,8 @@ def do_plot(self, data_plot, **backend_kwargs):
                     if len(wfs) > dp.max_spikes_per_unit:
                         random_idxs = np.random.permutation(len(wfs))[:dp.max_spikes_per_unit]
                         wfs = wfs[random_idxs][:, :, chan_inds]
+                    else:
+                        wfs = wfs[:, :, chan_inds]
```

Comment:
You whouls do `elif dp.max_spikes_per_unit is not None and len(wfs) > dp.max_spikes_per_unit:`

Reply:
it's the same, easier to read this way